post-commit

```diff
@@ -4,18 +4,20 @@
 # destination of the final changelog file
 OUTPUT_FILE=CHANGELOG.md
 
+# paste the source url of your git repository
+REPO_URL=https://github.com/JMAConsulting/auto-changelog-hook
+
 # generate the changelog
 # Change the repo link here to the one for the extension you are installing this in
-git --no-pager log --no-merges --format="%ci [%s](https://github.com/JMAConsulting/auto-changelog-hook/commit/%H) Author: %aN Committer: %cN" > $OUTPUT_FILE
+git --no-pager log --no-merges --format="%n #### %ai [%s](${REPO_URL}/auto-changelog-hook/commit/%H) <br> %aN (committed by %cN)<br><br>" > $OUTPUT_FILE
```

Comment:
Please change to:
git --no-pager log --no-merges --format="%ci [%s](${REPO_URL}/auto-changelog-hook/commit/%H) Author: %aN Committer: %cN" > $OUTPUT_FILE

Reply:
Done